### DISCUSSION TITLE
Send mail whit phpmailer

### DISCUSSION THREAD

**Discussion by Fevimo44** [PHPMailer/PHPMailer#2322] [General]
Because when I send an email with phpmailer, it looks perfectly on the mobile and tablet and yet on the laptop appears the header and foot unconfigured. . . . I'll thank you for yourHe thanks themI'll thank you for your feedback.I'd appreciate your feedback.

**Comment by Synchro** [comment]
Unfortunately this is life with email. Client apps vary far more than web browsers do, and you will run into inconsistencies everywhere, especially in gmail and outlook which are especially terrible.

It's also outside PHPMailer's control, as it just sends what you provide. You need to read up on formatting html for email.
